lua/astrocommunity/pack/typescript/init.lua

```diff
@@ -105,6 +88,16 @@ return {
       "AstroNvim/astrolsp",
       ---@diagnostic disable: missing-fields
       opts = {
+        autocmds = {
+          eslint_fix_on_save = {
+            cond = function(client) return client.name == "eslint" and vim.fn.exists ":EslintFixAll" > 0 end,
```

Comment:
```suggestion
            cond = function() return vim.fn.exists ":EslintFixAll" > 0 end,
```

Maybe we should just simplify it to this @Uzaaft what do you think?

Reply:
I have zero opinions about this 🙊

Reply:
Ok, let's just leave it as is